test/supp/drd-umf_test-ipc.supp

```diff
@@ -5,3 +5,13 @@
    fun:pthread_cond_destroy@*
    ...
 }
+
+{
+   [false-positive] Double check locking pattern in trackingOpenIpcHandle
+   drd:ConflictingAccess
+   ...
```

Comment:
why did you replace `fun:utils_atomic_load_acquire_ptr` with `...`?

You need to add the same suppression to the `helgrind-umf_test-ipc.supp` to solve Valgrind failure in CI.

Reply:
done